common-app-lib/src/cmr/common_app/api/launchpad_token_validation.clj

```diff
@@ -1,17 +1,61 @@
 (ns cmr.common-app.api.launchpad-token-validation
   "Validate Launchpad token."
   (:require
+   [cheshire.core :as json]
+   [clojure.data.codec.base64 :as base64]
+   [clojure.string :as string]
    [cmr.common-app.config :as config]
    [cmr.common.services.errors :as errors]
    [cmr.transmit.config :as transmit-config]))
 
+;; Note: Similar code exists at gov.nasa.echo.kernel.service.authentication
 (def URS_TOKEN_MAX_LENGTH 100)
 
+(defn- is-legacy-token?
+  "There are two uses cases captured by this test, the Legacy token, the
+   Heritage token (meaning a token made to behave like a legacy token but
+   prefixed with EDL-).
+   Note: Similar code exists at gov.nasa.echo.kernel.service.authentication."
+  [token]
+  (or (string/starts-with? token "EDL-")
+      (string/starts-with? token "Bearer EDL-")
```

Comment:
Change this line to `(string/starts-with? token "Bearer ")`, and move the line 22 size check to the first check.